xoxzo/cloudpy/cloudpy.py

```diff
@@ -0,0 +1,47 @@
+# xoxzo api libirary
+
+import requests
+
+
+class XoxzoCloud:
+    def __init__(self, sid, auth_token):
+        self.sid = sid
+        self.auth_token = auth_token
+        self.send_sms_last_respose = None
+        self.voice_palyback_last_respose = None
+
+    def send_sms(self, message, recipient, sender):
+        url = "https://api.xoxzo.com/sms/messages/"
```

Comment:
You can define a BASE_URL as a class variable, and then also make it possible to override it using environment variables. That make it possible for us to point the client to alternative endpoint, such as our development/local endpoint.

Also I think the sid and auth_token value you have to reference it from environment variables to avoid hardcoding it in the code itself.